### PR TITLE
Added a modifier allowsScrollViewBounces()

### PR DESCRIPTION
**Pull request by Kyome22**
In iOS, we want to control the behavior of `scrollView.bounces`.

### CODE REVIEW

File: Sources/WebUI/EnhancedWKWebView.swift [hunk 1]
```diff
@@ -24,6 +24,14 @@ class EnhancedWKWebView: WKWebView {
         }
     }
 
+    var allowsScrollViewBounces = true {
+        willSet {
+            #if os(iOS)
+            self.scrollView.bounces = newValue
+            #endif
+        }
+    }
```
Comment (ichiho-ojima) [inline Sources/WebUI/EnhancedWKWebView.swift:33]: Is it difficult to add UI tests?

File: Sources/WebUI/WebView.swift [hunk 2]
```diff
@@ -87,6 +88,16 @@ public struct WebView {
...
 
+    /// Sets value for scrollView.bounces to WebView.
+    /// - Parameters:
+    ///   - enabled: A Boolean value that controls whether the scroll view bounces past the edge of content and back again.
+    /// - Returns: WebView that controls whether the scroll view bounces past the edge of content and back again.
+    public func allowsScrollViewBounces(_ enabled: Bool) -> Self {
+        var modified = self
+        modified.allowsScrollViewBounces = enabled
+        return modified
+    }
```
Comment (ichiho-ojima) [inline Sources/WebUI/WebView.swift:99]: Is it difficult to add UI tests?

[the remark this answers is not in the document]
Comment (elmetal) [reply Sources/WebUI/WebView.swift:99]: I think a test of property values is enough and UITest is not necessary.

---
The default settings are as follows.

|property|value|
|-|-|
|`bounces`|true|
|`bouncesHorizontally`|false|
|`bouncesVertically`|false|

In this setting, the behavior depends on the size of the content and the screen size.

[the remark this answers is not in the document]
Comment (b1ackturtle) [reply Sources/WebUI/WebView.swift:99]: Thank you for organizing the points so clearly, @elmetal.
I understood that UI tests for this method are not necessary.

On another note, in `ExamplesUITests.swift`, we leave comments like the following for methods that do not have UI tests written.
```swift
// WebView.allowsInspectable(_:)
// Skip this test case as it is not possible to reproduce the operation.
```
 Should we write similar comments this time too?

[the remark this answers is not in the document]
Comment (Kyome22) [reply Sources/WebUI/WebView.swift:99]: @b1ackturtle @elmetal 
It certainly seems more consistent if we write a comment.

[the remark this answers is not in the document]
Comment (elmetal) [reply Sources/WebUI/WebView.swift:99]: @b1ackturtle @Kyome22 
It's a great idea to explicitly indicate skipping in comments!

[the remark this answers is not in the document]
Comment (Kyome22) [reply Sources/WebUI/WebView.swift:99]: [14c616a](https://github.com/cybozu/WebUI/pull/24/commits/14c616a2cdfe88933a4d40e800072c233c994fc9)
I wrote a comment.

**Review by b1ackturtle — APPROVED** [review]

**Review by elmetal — APPROVED** [review]

**Pull request merged**